dashboard/db/migrate/20160510192122_populate_hashed_email_in_users.rb

```diff
@@ -0,0 +1,18 @@
+# A migration that populates (if not already populated) a user's hashed_email
+# for users having an (non-empty) email address.
+
+class PopulateHashedEmailInUsers < ActiveRecord::Migration
+  def change
+    reversible do |direction|
```

Comment:
"old" does not imply "bad", I believe, at least in this case. They don't say that "up" and "down" are deprecated and harmful in all cases so I think we can use our judgement.

The change method is useful when there is shared logic between the up and down, but I don't think we should go out of our way to combine separate concerns if it means adding the moral equivalent of switch statements.